From 166a64f22b4085028cf9ef6f7a08fe7c0356daba Mon Sep 17 00:00:00 2001
From: Marc Kleine-Budde <mkl@pengutronix.de>
Date: Sat, 3 Apr 2010 14:42:29 +0200
Subject: [PATCH 1/2] tail: include sys/vfs.h

This patch fixes the following error:

tail.c:883: warning: implicit declaration of function 'fstatfs'

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 src/tail.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/tail.c b/src/tail.c
index 02c4a1a..d0bc957 100644
--- a/src/tail.c
+++ b/src/tail.c
@@ -57,6 +57,9 @@
 # if HAVE_SYS_STATFS_H
 #  include <sys/statfs.h>
 # endif
+# if HAVE_SYS_VFS_H
+#  include <sys/vfs.h>
+# endif
 #endif
 
 /* The official name of this program (e.g., no `g' prefix).  */
-- 
1.7.0.3

